src/styles.css

```
@@ -444,10 +444,127 @@ body{background:var(--bg);color:var(--ink);font-family:'Plus Jakarta Sans',sans-
+  background: #0A66C2;
+  border-color: #0A66C2;
+  box-shadow: 0 0 12px rgba(10, 102, 194, 0.5);
+}
+
+.social-link.twitter:hover {
+  background: #1DA1F2;
+  border-color: #1DA1F2;
+  box-shadow: 0 0 12px rgba(29, 161, 242, 0.5);
+}
+
+#backToTopBtn {
```

Stylelint (17.12.0)

[error] Expected id selector "`#backToTopBtn`" to be kebab-case (selector-id-pattern)

(selector-id-pattern)